app/views/quick_search/pages/home.html.erb

```diff
@@ -0,0 +1,50 @@
+<% title %>
```

Comment:
Is this intentional?  You're calling a helper and not drawing anything?

Reply:
This a straight port from upstream.. so yep.